src/main/java/ch/njol/skript/expressions/ExprName.java

```diff
@@ -89,19 +91,23 @@
 		"\t\t</ul>",
 		"\t</li>",
 		"</ul>"})
+@RequiredPlugins("Paper")
```

Comment:
Maybe this should be something like:
```suggestion
@RequiredPlugins("Paper to change player names")
```
Also, does this work on ALL paper versions since 1.9.4? If not, then it should also include like "Paper 1.x+ ...."

Reply:
I haven't found any docs for paper <1.12, do you know where I could possibly find them?

Reply:
https://shanebeestudios.com/docs/paper/